invesalius/gui/task_exporter.py

```diff
@@ -104,6 +105,20 @@ def __init__(self, parent):
         self.Update()
         self.SetAutoLayout(1)
 
+
+class CoordinateSpaceTransformHook(wx.FileDialogCustomizeHook):
+    def __init__(self):
+        super().__init__()
+        self.convertToWorld = None
+
+    def AddCustomControls(self, customizer):
+        self.checkbox = customizer.AddCheckBox(_("Convert to world coordinates"))
```

Comment:
Format text to regular color and not a blue as in a link. Does not look good in dark theme.